README.md

```diff
@@ -2,31 +2,369 @@
+)
+
+A relational database relies on **primary keys** and **foreign
+keys** to establish relationships between tables.
+
+- **Primary Key**: a column in a table with an identifier (ID) that uniquely
+  identifies one specific record, or row, in a table.
+- **Foreign Key**: a column in one table that refers to a primary key in
+  another table.
+
+Since a department has many employees, we can't use a single column in the `department` table
+to store the relationship.  Instead, we must update the `employee` table
+to store the employee's department id.
+
+The redesigned `department` table no longer contains the column for employees:
 
-## Topic 1
+| id  | name      | location   |
+|-----|-----------|------------|
+| 1   | Payroll   | Building A |
+| 2   | Marketing | Building B |
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+The redesigned `employee` table contains a new **foreign key** column `department_id` to store the **one-to-many** relationship:
 
-## Topic 2
+| id  | first_name | last_name  | salary | department_id |
+|-----|------------|------------|--------|---------------|
+| 1   | Kiran      | Willow     | 76000  | 2             |
+| 2   | Dani       | Elm        | 99000  | 2             |
+| 3   | Hao        | Pine       | 45000  | 2             |
+| 4   | Tal        | Oak        | 88000  | 1             |
+| 5   | Yuri       | Birch      | 150000 | 1             |
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+
+- We store a single integer in the `department_id` column, thus each employee works in one department.
+- Multiple rows in the `employee` table may have the same value in the `department_id` column,
+  thus a department may have multiple employees.
+- All columns contain atomic values.
+
+Now we can easily write the query to count the number of employees per department:
+
+```sql
+SELECT department_id, count(*)
+FROM employee
+GROUP BY department_id
+```
+
+The query result contains 2 rows since only 2 hotels have ratings of good or great:
+
+```text
+department_id   count 
+1               2
+2               3
+```
+
+
+What if we want to print the name of each department, along with the employee count?
+We need information from two tables: 
+(1) the count per `department_id` from the `employee` table, and
+(2) the name from the row in the `department` table that matches the `department_id` value
+from the `employee` table.
+That exact process of using a key in one table to identify a corresponding row
+in another table is what an SQL `JOIN` statement will do for us!  We will explore the
+`JOIN` statement in the next lesson.
+
+Other examples of **one-to-many** relationships:
+
+- An album has many songs.  Each song is on one album.
```

Comment:
Technically a song could be on multiple albums.
"Bohemian Rhapsody" for example is on the albums "A Night at the Opera" and "Greatest Hits"

Reply:
ok changed example to customer, purchase order ,etc